cli/lightning-cli.c

```diff
@@ -169,7 +169,7 @@ int main(int argc, char *argv[])
 {
 	int fd, i, off;
 	const char *method;
-	char *cmd, *resp, *idstr, *rpc_filename;
+	char *cmd, *resp, *idstr, *rpc_filename, *netname;
```

Comment:
This will break existing setups which do not set the (new) `--network` argument for `lightning-cli`, although a later commit makes this mostly work.